pkg/engine/jmespath/functions.go

```diff
@@ -550,6 +579,53 @@ func jpBase64Encode(arguments []interface{}) (interface{}, error) {
 	return base64.StdEncoding.EncodeToString([]byte(str.String())), nil
 }
 
+func jpTimeSince(arguments []interface{}) (interface{}, error) {
+	var err error
+	layout_, err := validateArg("", arguments, 0, reflect.String)
+	if err != nil {
+		return nil, err
+	}
+
+	layout := layout_.String()
+	if val, ok := layouts[layout]; ok {
+		layout = val
+	}
+
+	timestamp_1, err := validateArg("", arguments, 1, reflect.String)
+	if err != nil {
+		return nil, err
+	}
+
+	timestamp_2, err := validateArg("", arguments, 2, reflect.String)
+	if err != nil {
+		return nil, err
+	}
+
+	timestamp1 := timestamp_1.String()
+	timestamp2 := timestamp_2.String()
+
+	t1, err := time.Parse(layout, timestamp1)
+	if err != nil {
+		return nil, err
+	}
+
+	t2 := time.Now()
+
+	if timestamp2 != "" {
+		t2, err = time.Parse(layout, timestamp2)
+
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	if err != nil {
```

Comment:
Seems like an unnecessary error check here. No new value was assigned to `err`.

Reply:
Right, sorry I missed that.